lib/testing_api_onlyoffice_com/test_instance/main_page.rb

```diff
@@ -1,5 +1,6 @@
 require_relative 'main_page/community_server_api'
-require_relative 'main_page/document_builder_api'
+require_relative 'main_page/document_builder_api/document_builder_introduction'
```

Comment:
My experience with renaming folder and organizing structure advise me to change structure to this:

I think it's better to create single file `main_page/document_builder_api.rb` and list all requires in it
`cat lib/testing_api_onlyoffice_com/test_instance/main_page/document_builder_api.rb`
```
require_relative 'document_builder_api/documen_builder_integrating'
require_relative 'document_builder_api/document_builder_introduction'
```

and move left_side_navigation to subfolder like this
`tree lib/testing_api_onlyoffice_com/test_instance/main_page`

```
lib/testing_api_onlyoffice_com/test_instance/main_page
├── community_server_api.rb
├── document_builder_api
│   ├── document_builder_api_common
│   │   └── left_side_navigation.rb
│   ├── document_builder_integrating.rb
│   └── document_builder_introduction.rb
├── document_builder_api.rb
└── document_server_api.rb

2 directories, 6 files
```